apps/website/src/pages/docs/carousel.mdx

```diff
@@ -27,24 +25,22 @@ The currently shown slide can also be changed using the left/right arrow keys or
 
 ## Usage
 
+### Controlled State
+
 Carousels can be controlled externally by using the properties `activeSlideIndex` and `onSlideChange`.
```

Comment:
This section can be started by mentioning that the controlled state is not required since Carousel maintains the index/state by default. This can be worded similar to how it is done in [tabs#controlled-state](https://itwinui.bentley.com/docs/tabs#controlled-state).

E.g. `By default, the carousel maintain their own state. …`